snap/snapcraft.yaml

```diff
@@ -0,0 +1,60 @@
+name: matrix-appservice-slack
+base: core18
+version: git 
+summary: Node.js Matrix bridge for Slack
+description: |
+  A bridge between Matrix and Slack. Currently the bridge is in Beta.
+
+grade: stable 
+confinement: strict 
+
+apps:
+  matrix-appservice-slack: 
+    command: 'bin/matrix-appservice-slack-wrapper'
+    plugs: [network-bind, network]
+    daemon: simple
+parts:
+  nodejs-lts:
+    plugin: nil
+    override-build: |
+      NODE_VERSION=10.16.3
```

Comment:
Use 12.X if possible.

Reply:
No problem, I'll switch this to the latest v12 LTS